alembic/latiss/versions/53707815663e_change_primary_key_to_day_obs_seq_num.py

```diff
@@ -0,0 +1,210 @@
+    )
+    op.create_unique_constraint("un_exposure_exposure_id", "exposure", ["exposure_id"], schema="cdb_latiss")
+    op.add_column(
+        "exposure_flexdata",
+        sa.Column(
+            "day_obs",
+            sa.INTEGER().with_variant(mysql.INTEGER(), "mysql").with_variant(sa.INTEGER(), "postgresql"),
+            nullable=False,
+            comment="Day of observation.",
+        ),
+        schema="cdb_latiss",
+    )
+    op.add_column(
+        "exposure_flexdata",
+        sa.Column(
+            "seq_num",
+            sa.INTEGER().with_variant(mysql.INTEGER(), "mysql").with_variant(sa.INTEGER(), "postgresql"),
+            nullable=False,
+            comment="Sequence number.",
+        ),
+        schema="cdb_latiss",
+    )
+    op.drop_constraint("fk_obs_id", "exposure_flexdata", schema="cdb_latiss", type_="foreignkey")
+    op.create_foreign_key(
+        "fk_exposure_flexdata_day_obs_seq_num",
+        "exposure_flexdata",
+        "exposure",
+        ["day_obs", "seq_num"],
+        ["day_obs", "seq_num"],
+        source_schema="cdb_latiss",
+        referent_schema="cdb_latiss",
+    )
+    op.add_column(
+        "visit1_quicklook",
+        sa.Column(
+            "day_obs",
+            sa.INTEGER().with_variant(mysql.INTEGER(), "mysql").with_variant(sa.INTEGER(), "postgresql"),
+            nullable=False,
+            comment="Day of observation.",
+        ),
+        schema="cdb_latiss",
+    )
+    op.add_column(
+        "visit1_quicklook",
+        sa.Column(
+            "seq_num",
+            sa.INTEGER().with_variant(mysql.INTEGER(), "mysql").with_variant(sa.INTEGER(), "postgresql"),
+            nullable=False,
+            comment="Sequence number.",
+        ),
+        schema="cdb_latiss",
+    )
+    op.add_column(
+        "visit1_quicklook",
+        sa.Column(
+            "postisr_pixel_median",
+            sa.FLOAT().with_variant(mysql.FLOAT(), "mysql").with_variant(sa.FLOAT(), "postgresql"),
+            nullable=True,
+            comment="Median postISR pixel value.",
+        ),
+        schema="cdb_latiss",
+    )
+    op.create_foreign_key(
+        "fk_visit1_quicklook_day_obs_seq_num",
+        "visit1_quicklook",
+        "exposure",
+        ["day_obs", "seq_num"],
+        ["day_obs", "seq_num"],
+        source_schema="cdb_latiss",
+        referent_schema="cdb_latiss",
+    )
+    # ### end Alembic commands ###
+
+    # Added by hand: copy day_obs and seq_num into the ccdexposure,
+    # exposure_flexdata, and visit1_quicklook tables
+    # Extra commands to copy columns from the exposure table
+    # and mark the columns as non-null
+    pkey = {
+        "ccdexposure": "exposure_id",
+        "exposure_flexdata": "obs_id",
+        "visit1_quicklook": "visit_id",
+    }
+    the_schema = "cdb_latiss"
+    for destination_table in ("ccdexposure", "exposure_flexdata", "visit1_quicklook"):
+        for column in ("day_obs", "seq_num"):
+            op.execute(
```

Comment:
How would you downgrade from this? and did you add that part, but I'm just not seeing it?